prometheus exporter does not start on 0.16.0

**Describe the bug**
I am trying for the first time `-Dotel.metrics.exporter=prometheus`   with default interface and port, and am getting this error:
`java.lang.ClassNotFoundException: io.prometheus.client.exporter.HTTPServer`

Before that I was using `-Dotel.metrics.exporter=none`

Full stack trace:
```
otel-provider2_1     | [opentelemetry.auto.trace 2021-02-11 14:37:27:671 +0000] [main] INFO io.opentelemetry.javaagent.tooling.VersionLogger - opentelemetry-javaagent - version: 0.16.0
otel-provider2_1     | [opentelemetry.auto.trace 2021-02-11 14:37:27:908 +0000] [main] ERROR io.opentelemetry.javaagent.bootstrap.AgentInitializer - Throwable thrown while installing the agent
otel-provider2_1     | java.lang.reflect.InvocationTargetException
otel-provider2_1     | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
otel-provider2_1     | 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
otel-provider2_1     | 	at io.opentelemetry.javaagent.bootstrap.AgentInitializer.startAgent(AgentInitializer.java:65)
otel-provider2_1     | 	at io.opentelemetry.javaagent.bootstrap.AgentInitializer.initialize(AgentInitializer.java:51)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
otel-provider2_1     | 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
otel-provider2_1     | 	at io.opentelemetry.javaagent.OpenTelemetryAgent.agentmain(OpenTelemetryAgent.java:64)
otel-provider2_1     | 	at io.opentelemetry.javaagent.OpenTelemetryAgent.premain(OpenTelemetryAgent.java:51)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
otel-provider2_1     | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
otel-provider2_1     | 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
otel-provider2_1     | 	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(Unknown Source)
otel-provider2_1     | 	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallPremain(Unknown Source)
otel-provider2_1     | Caused by: java.lang.NoClassDefFoundError: io/prometheus/client/exporter/HTTPServer
otel-provider2_1     | 	at io.opentelemetry.sdk.autoconfigure.MetricExporterConfiguration.configurePrometheusMetrics(MetricExporterConfiguration.java:110)
otel-provider2_1     | 	at io.opentelemetry.sdk.autoconfigure.MetricExporterConfiguration.configureExporter(MetricExporterConfiguration.java:31)
otel-provider2_1     | 	at io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration.configureMeterProvider(OpenTelemetrySdkAutoConfiguration.java:48)
otel-provider2_1     | 	at io.opentelemetry.sdk.autoconfigure.OpenTelemetrySdkAutoConfiguration.initialize(OpenTelemetrySdkAutoConfiguration.java:31)
otel-provider2_1     | 	at io.opentelemetry.javaagent.tooling.OpenTelemetryInstaller.installAgentTracer(OpenTelemetryInstaller.java:36)
otel-provider2_1     | 	at io.opentelemetry.javaagent.tooling.OpenTelemetryInstaller.beforeByteBuddyAgent(OpenTelemetryInstaller.java:27)
otel-provider2_1     | 	at io.opentelemetry.javaagent.tooling.AgentInstaller.installComponentsBeforeByteBuddy(AgentInstaller.java:164)
otel-provider2_1     | 	at io.opentelemetry.javaagent.tooling.AgentInstaller.installBytebuddyAgent(AgentInstaller.java:98)
otel-provider2_1     | 	at io.opentelemetry.javaagent.tooling.AgentInstaller.installBytebuddyAgent(AgentInstaller.java:82)
otel-provider2_1     | 	... 18 more
otel-provider2_1     | Caused by: java.lang.ClassNotFoundException: io.prometheus.client.exporter.HTTPServer
otel-provider2_1     | 	at java.base/java.net.URLClassLoader.findClass(Unknown Source)
otel-provider2_1     | 	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
otel-provider2_1     | 	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
otel-provider2_1     | 	... 27 more
```

--- a/javaagent-tooling/javaagent-tooling.gradle
+++ b/javaagent-tooling/javaagent-tooling.gradle
@@ -45,7 +45,10 @@ dependencies {
   implementation deps.opentelemetryLogging
   implementation deps.opentelemetryOtlp
   implementation deps.opentelemetryOtlpMetrics
+
   implementation deps.opentelemetryPrometheus
+  implementation deps.prometheus
+
   implementation deps.opentelemetryZipkin
 
   implementation group: 'com.blogspot.mydailyjava', name: 'weak-lock-free', version: '0.15'

